src/Administration/Resources/app/administration/src/app/component/base/sw-card/sw-card.html.twig

```diff
@@ -57,6 +57,8 @@
             ></slot>
         </div>
 
+        <slot name="header-after"></slot>
```

Comment:
What is the purpose for this?

Because after the header only the toolbar should be visible. And then the content. So I would move it before the content.

Reply:
@jleifeld This is for the purpose of which I mentioned in this [comment](https://github.com/Shape-and-Shift/shopware-vue-app/pull/139#discussion_r1097015511) from the Admin SDK PR. If you have a better solution for it, I would love to implement it the other way.

Reply:
I understand your problem now. But I would suggest a different solution (explained in the comment)